Makefile

```
@@ -7,48 +7,40 @@ MAKEFLAGS += --no-builtin-rules
 
 BINARY := lazy-tmux
 
-.PHONY: help check build build-fzf build-all test test-race test-cov test-integration fmt fmt-check vet staticcheck golangci-lint lint tidy install clean dist dist-tui dist-fzf tag sandbox test-sup-versions docker-hub
+.PHONY: check build build-fzf build-all test test-race test-cov integration-test fmt golangci-lint lint install clean dist dist-tui dist-fzf tag sandbox test-sup-versions docker-hub vet
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)